src/devices/bus/ieee488/remote488.cpp

```diff
@@ -604,7 +604,7 @@ char remote488_device::recv_update(uint8_t& data)
 			m_rx_state = REM_RX_WAIT_CH;
 			break;
 		}
-		LOG("PARSE %02x %d->%d\n" , c , prev_state , m_rx_state);
+		//LOG("PARSE %02x %d->%d\n" , c , prev_state , m_rx_state);
```

Comment:
Why comment out these `LOG` lines?  That kind of defeats the purpose of configurable logging.  You'd be better off adding another bit to the `VERBOSE` mask to allow them to be enabled/disabled separately if they're too noisy.

See examples:
https://github.com/mamedev/mame/blob/84944bebfc78507b00723991a1bd768aef350e4c/src/devices/bus/ti99/internal/998board.cpp
https://github.com/mamedev/mame/blob/84944bebfc78507b00723991a1bd768aef350e4c/src/devices/cpu/m6800/m6801.cpp
https://github.com/mamedev/mame/blob/84944bebfc78507b00723991a1bd768aef350e4c/src/devices/machine/sdlc.cpp